dev/test-dev-env-setup.sh

```diff
@@ -21,7 +21,7 @@ export REPO_ROOT
 # Run the installation of the environment
 DEV_DIR=$REPO_ROOT/.venvs/mlflow-dev
 
-"$REPO_ROOT"/dev/dev-env-setup.sh -d "$DEV_DIR" -f
```

Comment:
The full installation is likely to fail.

Reply:
why does it fail ? it seems the CI job is canceled.

Reply:
You can see the installation logs. Pip downloads so many FLAML versions. This basically means "Hey, I can't resolve the dependencies", which happens when you install a large number of packages in one shot.

Reply:
Not testing `-f` isn't ideal but not sure if it's worth testing because you normally don't need it.